rhel-7-minimal.template

```diff
@@ -0,0 +1,172 @@
+sshpw --username=root --plaintext centos
```

Comment:
+1 to add the template.

Maybe it is time to rename this template. What's about _minishift-rhel-7.template_? Obviously the same for centos. IMO 'minimal' does not make much sense.